docs/integrations/destinations/snowflake.md

```diff
@@ -27,9 +27,15 @@ Note that Airbyte will create **permanent** tables. If you prefer to create tran
 
 ## Getting started
 
+### Requirements
+
+1. Active Snowflake warehouse
+2. A staging S3 or GCS bucket with credentials \(for the Cloud Storage Staging strategy\).
```

Comment:
I think we should remove this since everyone should use internal staging

Reply:
Yes thats correct @misteryeo